license.go

```diff
@@ -240,10 +251,10 @@ forUnknownFiles:
 
 	for name, licenses := range files {
 		if len(licenses) != 0 {
-			if len(licenses) > 1 || (licenses[0] != License(`Apache`) && licenses[0] != License(`Docs`) && licenses[0] != License(`Empty`) && licenses[0] != License(`Ignore`)) {
+			if len(licenses) > 1 || (licenses[0] != License(`Apache-2.0`) && licenses[0] != License(`Docs`) && licenses[0] != License(`Empty`) && licenses[0] != License(`Ignore`)) {
```

Comment:
Putting this in a constant might help readability and reduce the chances of future typos